angular.json

```diff
@@ -603,6 +603,43 @@
         }
       }
     },
+    "epd-visualization": {
+      "projectType": "library",
+      "root": "integration-libs/epd-visualization",
+      "sourceRoot": "integration-libs/epd-visualization/src",
+      "prefix": "lib",
```

Comment:
I am thinking about the better value for prefix. The term "lib" is too generic, isn't it?

Reply:
It looks like `cx` is used elsewhere. I will update it.